fire/cli/niv/_netoversigt.py

```diff
@@ -220,33 +183,80 @@ def path_to_origin(
     return None
 
 
-def analyser_subnet(net):
+def path_exists(net: dict[str, set[str]], start: str, origin: str) -> bool:
+    """
+    Bestem om net forbinder start til origin
+
+    Bruger breadth first search (BFS).
+    BFS Algoritme hugget fra: https://www.geeksforgeeks.org/breadth-first-search-or-bfs-for-a-graph/
+    """
+    visited = dict.fromkeys(net.keys(), None)
+
+    # Create a queue for BFS
+    q = deque()
+
+    # Mark the current node as visited and enqueue it
+    visited[start] = True
+    q.append(start)
+    while q:
+        currentNode = q.popleft()
+
+        # Get all adjacent vertices of the dequeued vertex
+        # If an adjacent has not been visited, then mark it visited and enqueue it
+        for neighbor in net[currentNode]:
+
+            # Skal ikke gøre andet end at returnere True hvis stien findes
+            if neighbor == origin:
+                return True
+
+            if not visited[neighbor]:
+                visited[neighbor] = True
+                q.append(neighbor)
+    return False
```

Comment:
Jeg synes ikke den her funktion bliver brugt nogle steder. Ligeledes lader den til at være så godt som identisk med `bfs()` der er indlejret i `analyser_subnet()`. Er det i virkeligheden noget efterladt kode fra en tidligere iteration?

Reply:
For at citere mig selv: ;) 

> Imidlertid ændrer https://github.com/SDFIdk/FIRE/commit/37d43894fff37f95b4fee0b377ecd0ba41f92aa7 på netgraf-funktionen, så hverken den gamle path_to_origin eller den nye path_exists nogensinde bliver kaldt. Hvis det ønskes kan jeg godt fjerne begge to funktioner i et nyt commit, for at gøre det hele mere "clean". 

Tænkte at det var noget der måske kunne bruges (siden det blev brugt før) men fint for mig hvis vi bare fjerner det.

Reply:
Gør det mere clean :)